### ISSUE TITLE
Image not being displayed on the people page

### ISSUE THREAD

**Issue by akankshat05** [PalisadoesFoundation/talawa-admin#809]
**Describe the bug**
Currently when we upload an image for admin, it is not being displayed on the site.
**To Reproduce**

1. Create an organization
2. Go to "Admins" from dashboard
3. Click on name of the member
4. Click on edit
5. Choose an image file for upload
6. Save changes

**Expected behavior**
After editing and saving the changes, they must be reflected on people page

**Video**

https://user-images.githubusercontent.com/72144454/227815566-922fb5eb-dd6b-4312-930f-f707ff80463c.mp4

**Potential internship candidates**
Please read this if you are planning to apply for a Palisadoes Foundation internship https://github.com/PalisadoesFoundation/talawa/issues/359

**Comment by akankshat05** [comment]
I would like to work on this issue. Thank you

**Comment by sarazaToussaint** [comment]
@akankshat05 for the image is displaying after upload

**Comment by sarazaToussaint** [comment]
@akankshat05 I'm sorry I made mistake. after editing the user details uploading the image is not displayed

**Comment by sarazaToussaint** [comment]
@akankshat05 I'm sorry I made mistake. after editing the user details uploading the image is not displayed

**Comment by sarazaToussaint** [comment]
@Sumitmaithani would please suggest me an issue to work on? I scared for the deadline

**Comment by Sumitmaithani** [comment]
@akankshat05 You can check out previous  issues where there is no activity for last 1-2 weeks or you can help those whose PR is not passing tests and find problem in solving issues

**Comment by palisadoes** [comment]
@Sumitmaithani you are right that this is part of a GSoC idea, however it is a precursor to a greater need to have better code reliability for existing and future features. We created issues to fix most of the obvious bugs, this one was missed. 

This long term goal wasn't specifically stated, though it was implied. I have updated the idea to reflect this.

![image](https://user-images.githubusercontent.com/16875803/227981887-a99d7444-6dba-40cc-bbef-f61ed5d8b3b0.png)

@sarazaToussaint I'll assign the issue to you
